ecommerce/courses/models.py

```diff
@@ -217,6 +217,7 @@ def create_or_update_seat(self, certificate_type, id_verification_required, pric
                 course_id
             )
 
+        import ipdb; ipdb.set_trace()
         stock_record.price_excl_tax = price
         stock_record.price_currency = settings.OSCAR_DEFAULT_CURRENCY
```

Comment:
definitively not, it was my approach without touching a big part of the ecommerce core code. Will talk to Bill de Rusha who is in charge of leading the new features development and will see if we can contribute to get multicurrency working in upstream